scheduler/src/cook/mesos/scheduler.clj

```diff
@@ -671,7 +678,7 @@
                                               db category->pending-jobs user->quota user->usage num-considerable))
               {:keys [matches failures]} (timers/time!
                                            handle-resource-offer!-match-duration
-                                           (match-offer-to-schedule fenzo (apply concat (vals category->considerable-jobs)) offers))
+                                           (match-offer-to-schedule fenzo (apply concat-vec (vals category->considerable-jobs)) offers))
```

Comment:
You can do `(reduce into (vals category->considerable-jobs))` instead of concat-vec